.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "recommendations": [
     "ms-azuretools.vscode-azurefunctions",
+    "ms-python.python",
```

Comment:
Could you create a new `.vscode` folder with these 4 files scoped for Python only inside the `src/python/AzureFunction.Http folder`? When a student then opens the workspace for this lesson, all of the settings and run commands will still work.

I will clean up this highest level `.vscode` folder to remove the lesson and language specific parts.

Reply:
I moved the `vscode` folder to `src/python/AzureFunctions.Http`, and I reverted the changes in the root `vscode`. Hope it looks okay now! :)